tests/Runner/TestSuiteSorterEmptyTestCaseTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PHPUnit\Runner;
+
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\TestSuite;
+
+class TestSuiteSorterEmptyTestCaseTest extends TestCase
```

Comment:
there is no `TestSuiteSorterEmptyTestCase` class, how we can have test for it ?

test here shall be part of `TestSuiteSorterTest` test class, it's one concrete, edge scenario to be tested, not completely other class